pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -295,37 +278,33 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
     }
 
     /**
-     * @notice Encodes a gradual swap fee update into the provided Pool state.
+     * @notice Encodes a gradual swap fee update into the Pool state in storage.
      * @param startTime - The timestamp when the swap fee change will begin.
      * @param endTime - The timestamp when the swap fee change will end (must be >= startTime).
      * @param startSwapFeePercentage - The starting value for the swap fee change.
      * @param endSwapFeePercentage - The ending value for the swap fee change. If the current timestamp >= endTime,
      * `getSwapFeePercentage()` will return this value.
-     * @return poolState - The modified Pool state with the updated swap fee data. It's the responsiblity of the caller
-     * to write this to storage so this value is persisted.
      */
     function _startGradualSwapFeeChange(
-        bytes32 poolState,
         uint256 startTime,
         uint256 endTime,
         uint256 startSwapFeePercentage,
         uint256 endSwapFeePercentage
-    ) internal returns (bytes32) {
+    ) internal {
         _validateSwapFeePercentage(startSwapFeePercentage);
         _validateSwapFeePercentage(endSwapFeePercentage);
 
         startTime = GradualValueChange.resolveStartTime(startTime, endTime);
 
         emit GradualSwapFeeUpdateScheduled(startTime, endTime, startSwapFeePercentage, endSwapFeePercentage);
```

Comment:
I'd like to move this below the assignment to `_poolState` now we aren't returning it.